runtime/src/parachains.rs

```diff
@@ -53,6 +57,7 @@ decl_storage! {
 	}
 	add_extra_genesis {
 		config(parachains): Vec<(Id, Vec<u8>, Vec<u8>)>;
+		config(_phdata): PhantomData<T>;
```

Comment:
I got an alternate version https://github.com/paritytech/polkadot/commit/fbed5bf3f0db3faf6386483494a83964aff54682 . At the end it also include a phantom data in GenesisConfig. There is also the possibility to do it for Code and Heads (put initialization value in a field build expression).